english/references/references.md

```diff
@@ -6,4 +6,6 @@ Then while making a gradual motion with 3 fingers on your right hand, you say
 Because 'the Force' is strong in you, the Chief Marshmallow agrees. Then with the same motion you say "Be on your way".
 The Chief Marshmallow looks at you and waves you on saying ["Be on your way"](../marshmallow.md)
 
-Before you go, you decide to ask the Chief Marshmallow for HIS [references](../chief-history/chief-history.md).
\ No newline at end of file
+Before you go, you decide to ask the Chief Marshmallow for HIS [references](../chief-history/chief-history.md).
+
+You see that the Chief Marshmellow is looking a little pale, and note that Chief Marshmellow's really look their best when golden brown. So you ["Lunch for a lighter to make him golden brown."](../lunge-for-the-lighter/lunge-for-the-lighter.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.